content/docs/addons-shallow-renderer.md

```diff
@@ -42,27 +42,27 @@ const result = renderer.getRenderOutput();
 
 expect(result.type).toBe('div');
 expect(result.props.children).toEqual([
-  <span className="heading">Title</span>,
+  <span className="heading">Judul</span>,
   <Subcomponent foo="bar" />
 ]);
 ```
 
-Shallow testing currently has some limitations, namely not supporting refs.
+Saat ini _shallow testing_ masih memiliki beberapa keterbatasan, salah satunya belum mendukung _refs_.
 
-> Note:
+> Catatan:
 >
-> We also recommend checking out Enzyme's [Shallow Rendering API](http://airbnb.io/enzyme/docs/api/shallow.html). It provides a nicer higher-level API over the same functionality.
+> Kami juga menyarankan Anda untuk membaca Enzyme's [Shallow Rendering API](http://airbnb.io/enzyme/docs/api/shallow.html) yang menyediakan _higher-level_ API yang lebih baik untuk fungsionalitas serupa.
 
-## Reference {#reference}
+## Referensi {#reference}
 
 ### `shallowRenderer.render()` {#shallowrendererrender}
 
-You can think of the shallowRenderer as a "place" to render the component you're testing, and from which you can extract the component's output.
+Anda dapat menggambarkan `shallowRenderer` sebagai “tempat” _render_ suatu komponen yang sedang Anda uji, juga sebagai tempat di mana Anda bisa mengekstrak keluaran dari komponen tersebut.
```

Comment:
Anda dapat membayangkan `shallowRenderer` sebagai "tempat" untuk me-*render* suatu komponen yang sedang Anda uji...

What do you think? 😄

Reply:
sounds good to me 👍